### PR TITLE
completed add-images branch

### PR DESCRIPTION
**Pull request by SammyT13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,15 @@ When creating an application it is important to think about the organization of
...
-**Branch**: add-images
+**Branch**: add-images **(Completed)**
 
 - As a user, instead of seeing a box I can see a picture of a light switch in the "off" position.
 - As a user, when I click on the light switch it turns "on".
 - As a user, when the light switch is "off" I see a white light bulb.
 - As a user, when I turn the switch "on" I see a yellow light bulb.
+- **Walkthrough of User Stories Completed GIF**:
+
+<img src="http://g.recordit.co/nvUp0uTxCs.gif" width=300><br>
```
Comment (jamesquillen) [inline README.md:42]: Great use of GIFs for documentation.

File: src/App.css [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
 }
 
 .lightbulb {
-  border: 2px solid black;
+  margin-left: auto;
+  margin-right: auto;
   height: 200px;
   width: 200px;
-  margin: 1rem auto; 
+  background-size: contain;
```
Comment (jamesquillen) [inline src/App.css:34]: These two classes are almost identical. You could make one class that contains all the common properties and then another one that only contains the differences and then chain them together to get the right combination. Here is an article that talks about [chaining](https://frontstuff.io/should-you-chain-or-extend-css-classes).

**Review by jamesquillen — COMMENTED** [review]
Great job.

File: src/App.css (hunk 2 above)
Comment (SammyT13) [reply src/App.css:34]: I appreciate your feedback.  I will follow up, and see if I can refactor.

**Pull request merged**